### DISCUSSION TITLE
Errors occurred when I installed ColossalAI

### DISCUSSION THREAD

**Discussion by xxw11** [hpcaitech/ColossalAI#494] [Community | Q&A] [answered]
I upgrad ColossalAI in the following way
![image](https://user-images.githubusercontent.com/55141908/159467824-caf07f66-2834-403a-932a-fa94e5a94b3d.png)
Some errors occurred when I installed ColossalAI.
And I tried to install using an older version, but I still got the same errors.
[error.log](https://github.com/hpcaitech/ColossalAI/files/8323634/error.log)
I don't know what caused these errors.
Any help would be much appreciated!

**Comment by FrankLeeeee** [comment, marked as answer]
Hi, it looks like a mismatch between library versions. Can you tell me your CUDA/PyTorch versions and GPU model?

**Reply by xxw11** [reply]
cuda version：
![image](https://user-images.githubusercontent.com/55141908/159471186-cdb8dde8-c49e-405a-ab90-af64761c2afd.png)

pytorch version：
![image](https://user-images.githubusercontent.com/55141908/159470973-93babdcd-5da1-493f-8be5-7095a79aceb4.png)

gpu model:
![image](https://user-images.githubusercontent.com/55141908/159471320-a8f5d752-608d-47ac-92f3-bbbbea860f02.png)

**Reply by FrankLeeeee** [reply]
Hi, this means the CUDA version is old, can you update your CUDA toolkit? You should use 11.3 for PyTorch 1.10.

**Reply by FrankLeeeee** [reply]
This can be found in the error log "nvcc fatal unsupported gpu architecture 'compute_86'" just for your information :)

**Reply by xxw11** [reply]
After trying you are right, I need to set my CUDA environment variable to 11.3. I have now installed it successfully. thank you so much~
